### DISCUSSION TITLE
API - OPNSense Compatible

### DISCUSSION THREAD

**Discussion by marlontf** [jaredhendrickson13/pfsense-api#560] [Ideas]
Is there any possibility of porting this API for use with OPNSense?

**Comment by jaredhendrickson13** [comment]
Probably not. This package works by calling internal functions that are mostly specific to pfSense. Many of the functions won't exist or will be different on OPNsense. Supporting it would require a significant portion of the package to be rewritten, and likely result in many features having to be removed as well.

**Comment by marlontf** [comment]
I understand your perspective and appreciate your response! Thank you very much for that.
